Samples/ExtendedControlAndMetadata/EyeGazeAndBackgroundSegmentation/README.md

```diff
@@ -0,0 +1,245 @@
+# WinRT sample application getting and setting extended camera controls as well as extract frame metadata
+
+This folder contains sample projects for
+- **EyeGazeAndBackgroundSegmentation**: a C# UWP application to preview camera and to interact with its extended controls
+- **CameraKsPropertyHelper**: a C++/Winrt runtime component to interact with a [VideoDeviceController](https://docs.microsoft.com/en-us/uwp/api/windows.media.devices.videodevicecontroller?view=winrt-20348) to get and set extended camera controls via serialized/deserialized byte buffers. While this sample covers EyeGazeCorrection and BackgroundSegmentation, it can be extended to cover other controls using the same methodology. 
+This project also contains a helper method to extract and deserialize frame metata from a byte buffer. Similarly while it covers how to extract *```KSCAMERA_METADATA_BACKGROUNDSEGMENTATIONMASK```* metadata, it can be extended to cover other type of metadata using the same methodology.  
+
+## Requirements
+	
+	This sample is built using Visual Studio 2019 and [Windows SDK version 20348](https://developer.microsoft.com/en-US/windows/downloads/windows-10-sdk/#serverdev) at minimum and requires [Windows SDK version 21364](TBD) to interact with the *KSCAMERA_METADATA_BACKGROUNDSEGMENTATIONMASK* metadata and the KSCAMERA_EXTENDEDPROP_BACKGROUNDSEGMENTATION_MASK capability for the KSPROPERTY_CAMERACONTROL_EXTENDED_BACKGROUNDSEGMENTATION control.
```

Comment:
>20348 [](http://example.com/codeflow?start=72&length=5)

This version is much higher than the intro .md file lists

Reply:
yep, this is the SDK that contains the bits needed

Reply:
You resolved won't fix but I see the number was lowered?